src/Tests.package/OSPlatformTest.class/instance/testNewFileInWorkingDirectory.st

```diff
@@ -1,8 +1,8 @@
 testing
 testNewFileInWorkingDirectory
 	"This is a test to make sure new files are created in the current working directory"
-| file |
+	| file |
 	file := File named: 'asd.txt'.
 	file writeStream close.
-self assert: file name asPath parent equals: OSPlatform current currentWorkingDirectoryPath asPath .
-file delete 
\ No newline at end of file
+	self assert: (FileSystem disk resolve: file name asPath parent) equals: OSPlatform current currentWorkingDirectoryPath asPath.
```

Comment:
Shouldn't this test only check that the just created file exists in the current directory?

```smalltalk
self assert: (FileSystem workingDirectory / 'asd.txt') exists
```